src/components/__tests__/reusables/SquareButton.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { render } from '@/utils/test-utils';
+import SquareButton from '@/components/reusables/SquareButton';
+
+describe('SquareButton component', () => {
+  test('matches the snapshot', () => {
+    const { asFragment } = render(
+      <SquareButton name={''} destination={'services'} />
```

Comment:
btw, it is good practice to add id to React components, and then in test you can assert that the element exists in document by using getElementById.